message-parser/requirements.txt

```diff
@@ -1,3 +1,34 @@
-grpcio>=1.29.0
-protobuf>=3.17.3
+#
```

Comment:
I think we need to keep the u/s requirements.txt file untouched and move all our files into a cachito directory.. that way both the u/s and d/s changes can persist in the repo

Reply:
Done